html2docx.py

```diff
@@ -6,10 +6,39 @@
 from docx import Document
 from docx.text.paragraph import Paragraph
 from docx.text.run import Run
+from tinycss2 import parse_one_declaration
+from tinycss2.ast import IdentToken
 
 WHITESPACE_RE = re.compile(r"\s+")
 
 
+def attr_to_css(attr: Tuple[str, Optional[str]]) -> Optional[Tuple[str, str]]:
+    if attr[0] == "style":
```

Comment:
I think tuple unpacking would aid readability. 0 and 1 are a bit opaque. Something like:

```diff
diff --git a/html2docx.py b/html2docx.py
index 9d20679..919d6d5 100644
--- a/html2docx.py
+++ b/html2docx.py
@@ -13,8 +13,9 @@ WHITESPACE_RE = re.compile(r"\s+")


 def attr_to_css(attr: Tuple[str, Optional[str]]) -> Optional[Tuple[str, str]]:
-    if attr[0] == "style":
-        declaration = parse_one_declaration(attr[1])
+    name, value = attr
+    if name == "style":
+        declaration = parse_one_declaration(value)
         for value in declaration.value:
             if isinstance(value, IdentToken):
                 return declaration.lower_name, value.lower_value
```